### PR TITLE
op-guide: update run execution path

### PR DESCRIPTION
**Pull request by siddontang**
@shenli @iamxy

### CODE REVIEW

File: op-guide/binary-deployment.md [hunk 1]
```diff
@@ -70,48 +70,48 @@ cd tidb-latest-linux-amd64
...
 
 2. 在每个节点启动 TiKV.
 
     ```bash
-    tikv-server -S raftkv \
-                -I 1 \
-                --pd 192.168.199.113:2379,192.168.199.114:2379,192.168.199.115:2379 \
-                --addr 192.168.199.113:20160 \
-                -s tikv1
+    ./tikv-server -S raftkv \
```
Comment (ngaut) [inline op-guide/binary-deployment.md]: without bin path ?

Comment (siddontang) [inline op-guide/binary-deployment.md]: my fault.

**Comment by siddontang** [comment]
PTAL @ngaut

**Comment by shenli** [comment]
LGTM

**Comment by iamxy** [comment]
LGTM

**Pull request merged**